src/components/widgets/Uenigboks.css

```diff
@@ -3,7 +3,7 @@
     flex: 1;
     align-content: stretch;
     align-items: flex-start;
-    padding: 0.3rem;
+    padding: 0.3rem 0;
```

Comment:
For å fikse feil med innrykk når `FormRow__Padder` er i bruk:
<img width="578" alt="image" src="https://user-images.githubusercontent.com/136872/63170474-a119c400-c039-11e9-92c2-726908fe67ff.png">